packages/webcomponents/package.json

```
@@ -143,7 +143,7 @@
   },
   "//": "TODO: make all of these optional dependencies, simple mode doesn't require",
   "dependencies": {
-    "@builder.io/react": "^2.0.4",
+    "@builder.io/react": "^2.0.13",
     "@builder.io/sdk": "^1.1.29",
```

```suggestion
    "@builder.io/sdk": "^1.1.30",
```
should we bump this up as well?

---

yep, will do. react sdk also uses 1.1.30 now.

---

done.